diff -up falkon-3.2.0/linux/applications/org.kde.falkon.desktop.2~ falkon-3.2.0/linux/applications/org.kde.falkon.desktop
--- falkon-3.2.0/linux/applications/org.kde.falkon.desktop.2~	2022-02-02 04:57:52.939688977 +0100
+++ falkon-3.2.0/linux/applications/org.kde.falkon.desktop	2022-02-02 04:58:12.154789643 +0100
@@ -36,7 +36,7 @@ Name[zh_CN]=Falkon
 Name[zh_TW]=Falkon
 Type=Application
 Icon=falkon
-Categories=Network;WebBrowser;
+Categories=KDE;Qt;X-MandrivaLinux-CrossDesktop;Network;WebBrowser;
 Comment=A fast and secure web browser
 Comment[ar]=متصفّح وبّ سريع وآمن
 Comment[az]=Sürətli və təhlükəsiz veb bələdçi
